tsconfig.json

```diff
@@ -9,7 +9,7 @@
 		"outDir": "_build/",
 		"removeComments": false,
 		"sourceMap": true,
-		"target": "es6",
+		"target": "es5",
```

Comment:
This was because we are now compiling a file that's run in the app `registerLoader`, and Uglify doesn't like es6.